pet_wellness_app1/app/assets/stylesheets/application.scss

```
@@ -14,3 +14,21 @@
  *= require_self
  */
  @import 'bootstrap';
+
+ .block {
+     background-color: grey;
+     margin: auto;
+     margin-top: 20px;
+     padding: auto;
+ }
+ .hero {
+     background-color: lightgrey;
+ }
+
+ .landing-cards {
+     margin-top: 20px;
```

This would be an easier way to write all 4 of these margin styles. When there are only two values for margin it uses them as top/bottom right/left
```suggestion
     margin: 20px auto;
```